batch_notification_processor/comms.py

```diff
@@ -71,3 +66,21 @@ def generate_hmac_signature(self, request_body: dict) -> str:
             msg=bytes(json.dumps(request_body), 'ASCII'),
             digestmod=hashlib.sha256
         ).hexdigest()
+
+    def send_get_request(self, url, headers, request_body):
```

Comment:
What do we gain by adding these two functions? They don't add any functionality that the `requests.get` | `requests.post` functions already give us.